### PR TITLE
add new section -USB Device Sharing

### PR DESCRIPTION
**Pull request by ailun258**
add new section into getting started guide so that people know how to share USB keyboard and mouse for UOS.

Signed-off-by: ailun258 <ailin.yang@intel.com>

### CODE REVIEW

**Comment by ailun258** [comment]
@dbkinder  waiting for you merge

**Comment by dongyaozu** [comment]
@ywan170 can u have a review？

**Comment by dongyaozu** [comment]
@dbkinder can u review/approve?

File: doc/getting-started/apl-nuc.rst [hunk 1]
```diff
@@ -364,6 +364,74 @@ For hugeTLB to work, you'll need to reserve huge pages:
...
      large enough for your usage.
 
+USB Device Sharing
+==========================================
+
+The ACRN hypervisor supports USB device sharing.  Suppose you have
+two keyboards and mice connected to your device, one keyboard and 
+mouse set for the SOS, and the other set for the UOS.  
+
+1. Boot the SOS and plug in the two keyboards and two mice
```
Comment (dbkinder) [inline doc/getting-started/apl-nuc.rst]: It still bothers me that you say to plug in all four devices here, and then two steps later you say to plug them in one by one.  How about adding a note as part of this first step:

    .. note::  You may need to plug one device in and run ``dmesg`` to display
       the device info, then plug in the second device and run ``dmesg`` again, 
       until you have all devices plugged in and have noted the device info
       specific for each device.

**Comment by ailun258** [comment]
@dbkinder please help check again !

**Pull request closed without merging**